app/controllers/renewals_controller.rb

```diff
@@ -78,17 +78,10 @@ def bulk_renewal_summary_flash(response, type)
     end
 
     def renewal_attempt_report(renewal_response)
-      if renewal_response.respond_to?(:each)
-        non_renewal_reason = error_prompt renewal_response[:error_message]
-        return renewal_response[:renewal].bib_summary + (non_renewal_reason || '')
+      if renewal_response[:sirsi_response].present?
+        return "#{renewal_response[:renewal].bib_summary} #{tag(:br)} #{renewal_response[:sirsi_response]}"
```

Comment:
```suggestion
        return "#{renewal_response[:renewal].bib_summary} #{tag(:br)} Denied: #{renewal_response[:sirsi_response]}"
```

Reply:
I thought I already committed this weird